usecases/traverser/grouper/merge_group.go

```diff
@@ -268,7 +268,7 @@ func mergeReferenceProps(in []interface{}) ([]interface{}, error) {
 				continue
 			}
 
-			id, ok := asRef.Fields["uuid"]
+			id, ok := asRef.Fields["id"]
 			if !ok {
 				return nil, fmt.Errorf("found a search.LocalRef, but 'uuid' field is missing: %#v", asRef)
```

Comment:
This potentially user-facing Error message should be changed from `'uuid' -> 'id'`

Reply:
ok, I'll change that